sonar-iac-plugin/src/main/resources/static/documentation.md

```diff
@@ -1,22 +1,23 @@
 ---
-title: Infrastructure-as-Code
+title: Terraform/CloudFormation
 key: iac
 ---
 
 <!-- static -->
 <!-- update_center:iac -->
 <!-- /static -->
 
-
-
 ## Language-Specific Properties
 
-Discover and update the language-specific [properties](/analysis/analysis-parameters/)
+Discover and update the CloudFormation [properties](/analysis/analysis-parameters/) **<!-- sonarcloud -->Project <!-- /sonarcloud -->[Administration > General Settings > Cloudformation](/#sonarqube-admin#/admin/settings?category=cloudformation)**
```

Comment:
I suggest:

`Discover and update the CloudFormation [properties](/analysis/analysis-parameters/) in: ...`

to follow what we have for other languages.

Also:
Administration > General Settings > Cloudformation
=>
Administration > General Settings > CloudFormation

Reply:
There is just a missing " in:" in the sentence.